build_matrix.yml

```diff
@@ -55,6 +55,11 @@ matrix:
       path: "packages/nodejs"
       variations:
         - name: "nodejs"
+      extra_tests:
+        diagnose:
+          - git submodule init
+          - git submodule update
+          - LANGUAGE=nodejs test/integration/diagnose/bin/test
```

Comment:
@jeffkreeftmeijer I've rebased this branch on the build matrix. I've added the diagnose tests as an "extra test" for the node.js package so it gets run for every node.js version as well.